src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+
+    {person.age !== undefined ? (
+      <p className="Person__age">I am {person.age}</p>
+    ) : null}
+
+    <p className="Person__partner">
+      {person.isMarried === false
```

Comment:
The same, don't compare with boolean variables
https://developer.mozilla.org/en-US/docs/Glossary/Falsy
https://developer.mozilla.org/en-US/docs/Glossary/Truthy